docs/DeveloperGuide.md

```diff
@@ -295,31 +302,33 @@ having to edit the mark and unmark feature difficult.
 
 &nbsp;
 
-### 2.6 Display feature
-#### 2.6.1 Current implementation
+### 3.6 Display feature
+This feature involves displaying the shopping list and budget details to the user.
+#### 3.6.1 Current implementation
 
-The display feature is implemented using a <code>ListCommand</code> class which extends the <code>Command</code> class. 
+The display feature is implemented using a <code>DisplayCommand</code> class which extends the <code>Command</code> 
+class. 
  
 The <code>Duke</code> class first receives user input from the <code>Ui</code> object before it creates a 
 <code>Parser</code> object and calls its <code>parseCommand</code> function to instantiate a 
-<code>ListCommand</code> object based on that user input.
+<code>DisplayCommand</code> object based on that user input.
 
-The <code>Duke</code> class then calls the <code>execute</code> method of the <code>ListCommand</code> object.
+The <code>Duke</code> class then calls the <code>execute</code> method of the <code>DisplayCommand</code> object.
 This method makes a call to the <code>getTotalCost</code> method of the <code>ShoppingList</code> object to find the 
 cost of the items. It then calls the <code>getAmount</code> and <code>getRemainingBudget</code> methods of the 
 <code>Budget</code> object to find the current budget and the remaining budget. The results are then printed to console.
 
 The following sequence diagrams below show how the display feature works. Note the <code>Ui</code> class is
 omitted to emphasise the other classes:
 
-![alt text](images/ListFeature.png)
+![alt text](images/List.png)
 
-![alt text](images/ListFeature_SD.png)
+![alt text](images/List_SD.png)
 
```

Comment:
These two diagrams are not appearing in the developer guide for some reason. Might have to fix that.

Reply:
Oh yea, forgot that I changed the names of the diagrams, thanks!